src/LogicTasks/Semantics/Resolve.hs

```diff
@@ -199,8 +225,13 @@ completeGrade ResolutionInst{..} sol =
                             else refuse $ indent $ translate $ do
                                    german "Die Leere Klausel wurde nicht korrekt abgeleitet."
                                    english "The Empty clause was not derived correctly."
+
```

Comment:
Gibt es hier jetzt eventuell noch Redundanzen? Also insbesondere das Feedback basierend auf der Fallunterscheidung auf `applySteps clauses steps` in den Zeilen (nach "neuer Zählung") 218-227, gibt es konzeptionell anderes/zusätzliches Wissen aus als schon die Zeilen nach `unless showFeedbackOnPartialGrade` in `completeGrade`?

Reply:
Ja, gab es. Ich habe es angepasst.